September Webinar 2024 (#2534)
FlowFuse/website: src/webinars/2024/operationalizing-node-red-for-the-enterprise.md

<!--more-->

When it comes to quick proto-typing and small-scale deployments, Node-RED is an ideal open-source tool. With its low-code simplicity, unparalleled protocol connectivity, and backing of a thriving community, it is often the platform of choice for industrial engineers looking to deploy their own solutions without relying too much on IT departments. 

While this bottom-up approach brings agility and innovation, it's crucial to ensure that your Node-RED deployments are enterprise-ready. This means having the right safeguards in place, adhering to compliance frameworks, and ensuring your systems are manageable at scale. 

In this webinar, we will address the key challenges of scaling Node-RED for the enterprise and introduce how FlowFuse can take your projects to the next level. 

ZJ van de Weg, CEO of FlowFuse will cover
 - How to maintain control and visibility of your Node-RED deployments. 
 - Strategies for managing and scaling your Node-RED flows
 - How to secure your Node-RED environment with features like HTTPS, SSO, and regular updates.

> Rob Marcer will be joining ZJ to share some real-world examples of how FlowFuse Customers were able to quickly and easily manage their enterprise deployments.  

Reviewer: ```suggestion
Rob Marcer, Customer Success Manager at FlowFuse, will be joining ZJ to share real-world examples of how FlowFuse customers are quickly and easily managing their enterprise deployments.  
```
